scripts/process/process.sparks.R

```diff
@@ -50,24 +25,32 @@ grab_clip_rect <- function(vals, flood){
   }
 }
 
-# Should rename at some point. Not always going to be discharge.
-process.flood_sparks <- function(viz = as.viz('flood-sparks')){
+process.sparks <- function(viz = as.viz('sparks')){
   library(dplyr)
   depends <- readDepends(viz)
   checkRequired(depends, c("gage-data", "sites", "nws-data"))
-  
   sites <- depends[["sites"]]
+  gage_data <- depends[["gage-data"]]$timestep_q
+  site_nils <- depends[["gage-data"]]$gage_mask_vals
```

Comment:
This thing looks like: 
```
Browse[2]> str(site_nils)
 Named num [1:24] 4.75 4.92 1.53 2.36 11.2 ...
 - attr(*, "names")= chr [1:24] "50024950" "50027600" "50031200" "50035000" ...
```
So you can grab the nil value like: `mask_val <- site_nils[[sites$site_no[1]]]`